jenkins-centos7-atomicapp/Nulecule

```diff
@@ -0,0 +1,32 @@
+---
+specversion: 0.0.2
+id: jenkins-atomicapp
+
+metadata:
+  name: Jenkins App
+  appversion: 1.635
+  description: Jenkins is an open-source continuous integration server.
+graph:
+  - name: jenkins
+    params:
+      - name: image
+        description: Jenkins image to be used
+        default: kumarpraveen/centos-jenkins:latest
```

Comment:
could you check if you can use this: https://hub.docker.com/r/openshift/jenkins-1-centos7/